src/services/redis/storeAttOrMidToRedis.ts

```
@@ -1,8 +1,10 @@
+import { logger } from "../../utils/logger.js";
 import * as cheerio from "cheerio";
 import { Academic } from "../student.utils/Academic.js";
 import { getClient } from "./getRedisClient.js";
 import { updateAttendanceStat, updateMidMarkStat } from "../../db/student_stats.model.js";
 import { getStudentCached } from "./utils.js";
+import { logger } from "../../utils/logger.js";
```

Biome (2.3.13)

[error] Shouldn't redeclare 'logger'. Consider to delete it or rename it.

'logger' is defined here:


(lint/suspicious/noRedeclare)